source/tv/phantombot/panel/PanelUser/PanelUserHandler.java

```diff
@@ -0,0 +1,470 @@
+package tv.phantombot.panel.PanelUser;
+
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import org.json.JSONStringer;
+import com.gmt2001.datastore.DataStore;
+import tv.phantombot.PhantomBot;
+
+/**
+ * {@link PanelUser Panel user} management for the web panel and websocket
+ *
+ * @author Sartharon
+ */
+public final class PanelUserHandler {
+    /**
+     * Database table suffix for the {@link PanelUser panel users}
+     */
+    protected static final String PANELUSERTABLE = "panelUsers";
```

Comment:
Codacy: Avoid protected fields in a final class.  Change to private or package access.